src/avfoundation.cs

```diff
@@ -12461,6 +12726,21 @@ interface AVSampleBufferDisplayLayer {
 		[Notification]
 		NSString RequiresFlushToResumeDecodingDidChangeNotification { get; }
 
+		[TV (14, 5), Watch (7, 4), Mac (11, 3), iOS (14, 5)]
+		[Field ("AVSampleBufferDisplayLayerOutputObscuredDueToInsufficientExternalProtectionDidChangeNotification")]
```

Comment:
OutputObscuredDueToTextGoingBeyondScreenSoBiggerScreenRequested

Reply:
hmm what do you mean? Are you suggesting I rename the NSString var to this?

Reply:
j/k about the long name ;-)